### PR TITLE
Refactor env vars names

### PR DESCRIPTION
**Pull request by selankon**
Use

DEGEN_CONTRACT_ADDRESS -> VOCDONI_COMMUNITYHUBADDRESS 
RESULTS_CONTRACT_ADDRESS -> VOCDONI_COMMUNITYRESULTSADDRESS

### CODE REVIEW

File: webapp/vite.config.ts [hunk 1]
```diff
@@ -14,8 +14,8 @@ export default defineConfig({
   },
   define: {
     'import.meta.env.APP_URL': JSON.stringify(process.env.APP_URL || ''),
-    'import.meta.env.DEGEN_CONTRACT_ADDRESS': JSON.stringify(process.env.DEGEN_CONTRACT_ADDRESS || '0xd4768df803c5a9eDA475159cfbBcF9c06F077c13'),
-    'import.meta.env.RESULTS_CONTRACT_ADDRESS': JSON.stringify(
+    'import.meta.env.VOCDONI_COMMUNITYHUBADDRESS': JSON.stringify(process.env.DEGEN_CONTRACT_ADDRESS || '0xd4768df803c5a9eDA475159cfbBcF9c06F077c13'),
```
Comment (lucasmenendez) [inline webapp/vite.config.ts:17]: change also `process.env.DEGEN_CONTRACT_ADDRESS`

File: webapp/vite.config.ts [hunk 2]
```diff
@@ -14,8 +14,8 @@ export default defineConfig({
...
+    'import.meta.env.VOCDONI_COMMUNITYRESULTSADDRESS': JSON.stringify(
```
Comment (lucasmenendez) [inline webapp/vite.config.ts:18]: same

Comment (lucasmenendez) [reply webapp/vite.config.ts:18]: and populate the changes in the files that import `import.meta.env.DEGEN_CONTRACT_ADDRESS` and `import.meta.env.RESULTS_CONTRACT_ADDRESS` (old names)

**Review by lucasmenendez — CHANGES_REQUESTED** [review]

File: webapp/vite.config.ts (hunk 2 above)
Comment (selankon) [reply webapp/vite.config.ts:18]: Done

**Pull request merged**